extensions/3DTILES_content_voxels/README.md

```diff
@@ -93,13 +93,23 @@ Axis|Coordinate|Positive Direction
 
 ![Cylinder Coordinates](figures/cylinder-coordinates.png)
 
-The figure below shows `"dimensions": [8, 8, 8]` for each shape type:
+These conventions align with how implicit tile coordinates defined in [Implicit Tiling](../../specification/ImplicitTiling/). The figure below shows `"dimensions": [8, 8, 8]` for each shape type:
 
 |Box|Region|Cylinder|
 | ------------- | ------------- | ------------- |
 |![Box Voxel Grid](figures/box.png)|![Region Voxel Grid](figures/sphere.png)|![Cylinder Voxel Grid](figures/cylinder.png)|
 
-Dimensions must be nonzero. Elements are laid out in memory first-axis-contiguous, e.g. for boxes, x data is contiguous.
+> **Implementation Note**: glTF and 3D Tiles use different up-axes, which results in different expectations for voxel **data order**. Although both use an `[x, y, z]` ordering, the meanings of the `y` and `z` axes are not one-to-one for **box** or **cylinder** voxels.
+>
+> For **box** voxels, the grid's height corresponds to `y` in glTF, but must ultimately align with `z` in 3D Tiles. If the dimensions in `3DTILES_content_voxels` are `[width, height, depth]`, then the dimensions in `EXT_primitive_voxels` **MUST** be `[width, depth, height]`. Furthermore, if a **box** voxel is located at `[x, y, z]` within 3D Tiles, it refers to the voxel at `[x, depth-z, y]` in the glTF. 
+>
+> (From the glTF's perspective, if its dimensions are `[width, height, depth]`, then a **box** voxel at `[x, y, z]` will be located at `[x, z, height-y]` in 3D Tiles.)
```

Comment:
Should it be `depth-1-z` and `height-1-y`?

Reply:
Oh good catch, thanks!